src/Compilers/CSharp/Portable/Utilities/ValueSetFactory.IntTC.cs

```diff
@@ -46,7 +58,7 @@ int INumericTC<int>.Next(int value)
 
             int INumericTC<int>.Prev(int value)
             {
-                Debug.Assert(value != int.MinValue);
+                Debug.Assert(value != (nonNegative ? 0 : int.MinValue));
```

Comment:
>(nonNegative ? 0 : int.MinValue) [](http://example.com/codeflow?start=38&length=32)

```MinValue```?

Reply:
MinValue is a private interface implementation

Reply:
> MinValue is a private interface implementation

It doesn't have to be private. And we can avoid code duplication.